src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
body don't have default paddings, and there is no need to set `box-sizing` to body (it's not inherited)